commit 1b2adaee621f520aa640669a35c089c448294e9e
Author: Alexander Tsoy <alexander@tsoy.me>
Date:   Sat Oct 17 20:39:10 2015 +0300

    Fix conflicting types errors in radius auth
    
    Was broken by commits 93a6fce2 and 55414b25

diff --git a/src/src/auths/call_radius.c b/src/src/auths/call_radius.c
index 2064ed2..1201078 100644
--- a/src/src/auths/call_radius.c
+++ b/src/src/auths/call_radius.c
@@ -60,10 +60,10 @@ Returns:   OK if authentication succeeded
 */
 
 int
-auth_call_radius(uschar *s, uschar **errptr)
+auth_call_radius(const uschar *s, uschar **errptr)
 {
 uschar *user;
-uschar *radius_args = s;
+const uschar *radius_args = s;
 int result;
 int sep = 0;
 
